Cpix/CpixDocument.cs

```diff
@@ -54,6 +55,48 @@ public string ContentId
 			}
 		}
 
+		/// <summary>
+		/// Gets or sets the version of this CPIX document. The value shall reference a published version of
+		/// the CPIX Guidelines and be structured as majorVersion.minorVersion (Example: '2.3'). If the CPIX
+		/// client knows that it doesn’t support all the features of a given CPIX version, it needs to behave
+		/// according to the recommendations of the API used to exchange the CPIX document.
+		/// </summary>
+		public string Version
+		{
+			get => VersionInternal;
+
```

Comment:
Extra space?

Reply:
Removed newline from between `get` and `set`.